doc/rst/technotes/gpu.rst

```diff
@@ -139,22 +139,27 @@ To enable GPU support set the environment variable ``CHPL_LOCALE_MODEL=gpu``
 before building Chapel.
 
 Chapel's build system will automatically try and deduce what type of GPU you
-have and where your installation of relevant runtime (e.g. CUDA or ROCM) are.
-If the type of GPU is not detected you may set ``CHPL_GPU_CODEGEN`` manually to
-either ``cuda`` (for NVIDIA GPUs) or ``rocm`` (for AMD GPUs). If the relevant
-runtime path is not automatically detected (or you would like to use a
-different installation) you may set ``CHPL_CUDA_PATH`` and/or
-``CHPL_ROCM_PATH``.
+have and where your installation of relevant runtime (e.g. CUDA or ROCm) are.
+If the type of GPU is not detected you may set ``CHPL_GPU`` manually to either
```

Comment:
If the type of GPU is not detected you may set ``CHPL_GPU`` manually to either ``nvidia`` or ``amd``

-->  

If the type of GPU is not detected you may set ``CHPL_GPU`` manually to either ``nvidia``, ``amd``, or ``cpu`` (see here [link to cpu section]).